pyproject.toml

```diff
@@ -23,10 +23,10 @@ classifiers = [
 
 dependencies = [
     "scikit-learn~=1.2",
-    "numpy >=1.11,<1.24",
-    "pandas~=1.4",
+    "numpy >=1.22, <1.27", # limited by Numba support
+    "pandas>=1.4",
     "joblib~=1.2",
-    "numba~=0.56",
+    "numba~=0.58",
```

Comment:
just in case numba releases a new version and anonymeter users would like to use it. Otherwise, we would need to release a new version of anonymeter each time a new version of numba is released.

Reply:
Makes sense. Thanks!